resources/arduino/devkit/blink.ino

```diff
@@ -0,0 +1,13 @@
+// the setup function runs once when you press reset or power the board
```

Comment:
TODO: @dilin-MS File sctructure under `resources/arduino/devkit/` will be better designed later.